modules/minimum-required-permissions-ipi-gcp.adoc

```diff
@@ -184,6 +184,7 @@ If your organization’s security policies require a more restrictive set of per
 [%collapsible]
 ====
 * `iam.roles.get`
+* `iam.serviceAccounts.signBlob`
```

Comment:
`signBlob` is not required for all installations--only when the installer is authenticated without a service account key, which is with Application Default Credentials. So perhaps we want to move this to a permissions section called `Required permissions when authenticating with Application Default Credentials`.

Reply:
There may be other methods of authenticating, besides "Application Default Credentials" which do not provide a key, so perhaps we want to call this `Required permissions when authenticating without a service account key"